### DISCUSSION TITLE
Tokensmith

### DISCUSSION THREAD

**Discussion by cl-fi** [creatorsand/.github#141] [Ideas]
## project
- 项目名称: TokenSmith

>描述: 一个无许可的期权平台。

### Intro
如何创造区块链上的期权产品一直是行业的一大难点，至今尚未出现真正成功的链上期权平台。现有的链上期权交易平台（如 rsyk finance）往往在代币激励结束后，面临严重的用户流失问题。

我们认为链上期权难以发展的主要原因有两个：
首先，期权作为一种金融衍生品，涉及immplied volatility、gamma等复杂概念，这些专业术语对链上用户来说较难理解和把握。其次，区块链的无许可（permissionless）特性特别适合发展长尾资产，但这些资产普遍波动性过高，难以适用传统期权交易模式。而如果仅做比特币、以太坊等主流加密货币的期权交易，又难以在流动性上与 Deribit 等中心化期权交易平台竞争。

因此，我们认为链上期权产品应该突破传统期权交易仅用于风险对冲的局限，而是要基于链上用户的交易习惯和区块链资产的独特特点进行创新，开创全新的应用场景和价值。就像 TokenSmith 开创的无许可衍生期权代币模式一样。

本项目与 Typus 的主要区别在于：
1. 无许可特性：TokenSmith 允许任何用户对任意资产创建担保看涨/看跌期权，无需平台审批，从而更好地服务长尾资产市场。
2. 代币化设计：创建的期权以代币形式存在，用户可以自由转让，并在任何 AMM 或 DeepBook 上进行交易，更符合链上用户的使用、交易习惯。

本项目主要应用场景：
1. 优化 Launchpad 激励机制：相比直接发放代币，期权激励方式更有利于项目的健康发展。
2. 改善项目空投策略：与直接空投项目代币相比，空投期权可以有效缓解卖压，降低对代币价格的负面影响。
3. ...

## Member
- Chenjie  github: https://github.com/web3CL
> 自我介绍&技术栈: DeFi 产品， Move 合约  

- Lianshu  github: https://github.com/lianshus
> 自我介绍&技术栈: 前端

**Comment by v1xingyue** [comment]
ai打分

[a comment by an automated account is not shown]
